pkg/apis/navigator/validation/cassandra.go

```diff
@@ -11,7 +12,23 @@ import (
 )
 
 func ValidateCassandraClusterNodePool(np *navigator.CassandraClusterNodePool, fldPath *field.Path) field.ErrorList {
-	return field.ErrorList{}
+	allErrs := field.ErrorList{}
+	if np.Seeds > np.Replicas {
+		allErrs = append(allErrs,
+			field.Invalid(
+				fldPath.Child("seeds"),
+				np.Seeds,
+				fmt.Sprintf("number of seeds cannot be greater than number of replicas (%d)", np.Replicas)),
+		)
+	}
+
+	if np.Seeds < 0 {
+		allErrs = append(allErrs,
+			field.Invalid(fldPath.Child("seeds"), np.Seeds, "number of seeds must be greater than or equal to 1"),
+		)
+	}
```

Comment:
The message says "must be greater than or equal to 1" but the test is `np.Seeds < 0`